docs/topics/vscode-extension-configuring-analysis.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-configuring-analysis_{context}']
+
+= Configuring a run configuration
+
+You can configure a number of separate run configurations to run against each of your projects. Each configuration is saved automatically as you work.
+
+.Prerequisites
+
+* Install `mta-cli`.
+* Projects imported into {VscodePluginShortName}.
+
+.Procedure
+
+. In *Extensions* view, click the *{ProductName}* icon (image:vs_MTA_extension_icon.png[MTA code extension]) on the Activity bar.
+. If you do not have the MTA CLI installed, click *Download* on the *Unable to find MTA CLI* window.
+. Click the *+* (plus sign) next to *{ProductName}* to add a run configuration.
+. Optional: Enter a new *Name*.
+. At a minimum, complete the following configuration fields:
+
+* *Input*: Click *Add* and enter the input files or directories or click *Open File Explorer* and select the directory.
```

Comment:
Check the UI. Is that one button (Add) or two? I think there is just the "Add" button but this sentence sounds like there are two buttons.